### DISCUSSION TITLE
Is RAC enabled using OraCdcSourceConnector?

### DISCUSSION THREAD

**Discussion by joontube** [averemee-si/oracdc#61] [Q&A] [answered]
RAC related functions are found in OraCdcLogMinerConnector but not in OraCdcSourceConnector.
Are they not supported?

**Comment by averemee-si** [comment, marked as answer]
Hi Seongjoon,

Thank you very much for your interest in oracdc!

No, this is impossible - OraCdcSourceConnector connector works at the level of logical objects, and they are the same for each RAC node, unlike redo threads processed by OraCdcLogMinerConnector.

Feel free to contact me if you have any further questions about oracdc, its configuration and usage.

Best regards,
Aleksei

**Reply by joontube** [reply]
thanks for the explanantion
